team.html

```diff
@@ -0,0 +1,80 @@
+<!doctype html>
+<html class="no-js" lang="">
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="x-ua-compatible" content="ie=edge">
+        <title>Team HINT 2.0</title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+
+        <!-- Google Webfonts -->
+        <link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet">
+
+        <link rel="stylesheet" href="static/css/normalize.css">
+        <link rel="stylesheet" href="static/css/team.css">
+        <script src="static/js/vendor/modernizr-2.8.3.min.js"></script>
+    </head>
+    <body>
+        <!--[if lt IE 8]>
+            <p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
+        <![endif]-->
+
+        <!-- Add your site or application content here -->
+        <div class="heading">
+            <h1>Team Hack In The North 2.0</h1>
+        </div>
+
+        <div class="container">
+          <div class="btn-group controls-holder">
+            <div class="btn filter active" data-filter="all">Show All</div>
+            <div class="btn filter" data-filter=".category-1">Sponsorship</div>
+            <div class="btn filter" data-filter=".category-2">Website</div>
+            <div class="btn filter" data-filter=".category-3">Onsite</div>
+          </div>
+        </div>
+
+        <div id="cardframe">
+
+            <div class="card mix category-1" data-myorder="1">
+                <img class="card-image" src="static/images/profile.jpg" />
+                <p>John Doe</p>
+            </div>
+
+            <div class="card mix category-1" data-myorder="2">
+                <img class="card-image" src="static/images/profile.jpg" />
+                <p>John Doe</p>
+            </div>
+
+            <div class="card mix category-2" data-myorder="3">
+                <img class="card-image" src="static/images/profile.jpg" />
+                <p>John Doe</p>
+            </div>
+
+            <div class="card mix category-2" data-myorder="4">
+                <img class="card-image" src="static/images/profile.jpg" />
+                <p>John Doe</p>
+            </div>
+
+
+
+            <div class="gap"></div>
+            <div class="gap"></div>
+        </div>
+
+        <script src="https://code.jquery.com/jquery-1.12.0.min.js"></script>
+        <script>window.jQuery || document.write('<script src="static/js/jquery.js"><\/script>')</script>
```

Comment:
The older version was included by default in the HTML5 boiler plate code. Now I have removed it. :sweat_smile: